app/validation/userValidation.js

```diff
@@ -0,0 +1,95 @@
+const Joi = require('joi');
+const { ROLE } = require('../utils/enum');
+
+module.exports = {
+  registerValidation: Joi.object().keys({
+    name: Joi.string().required().empty().messages({
+      'string.base': `name should be a type of 'text'`,
```

Comment:
When you give message so please start with capital later then small letter. 
like  Name should be a type of 'text'